libcxx/include/bitset

```diff
@@ -187,8 +187,8 @@ protected:
 
```

Comment:
The potential breaking is more concealed because these member typedefs are already private ([Godbolt link](https://godbolt.org/z/5fETsxfMq)). IIUC only non-conforming code that expects such ambiguity (via SFINAE or requires-expression on member types) will be made rejected ([Godbolt link](https://godbolt.org/z/cGhsWGqYs)).

I'll try to mention the changes in release notes.

Reply:
Ah, I missed that they were private. I think the release note is OK anyway. Thanks!